mvc/views/tag-helpers/intro.rst

```diff
@@ -1,8 +1,275 @@
+   :language: c#
+   :lines: 12-18
+   :dedent: 4
+
+Each of the "asp-" attributes has a value of "Email", but "Email" is not a string. In this context, "Email" is the C# model expression property for the ``RegisterViewModel``. 
+
+
+The Visual Studio editor helps you write **all** of the markup in the Tag Helper approach of the register form, while Visual Studio provides no help for most of the code in the HTML Helpers approach. `IntelliSense support for Tag Helpers`_ goes into detail on working with Tag Helpers in the Visual Studio editor.
+
+Tag Helpers compared to Web Server Controls
+-----------------------------------------------
+
+- Tag Helpers don't own the element they're associated with; they simply participate in the rendering of the element and content. ASP.NET `Web Server controls <https://msdn.microsoft.com/en-us/library/7698y1f0.aspx>`__ are declared and invoked on a page.
+
+- `Web Server controls <https://msdn.microsoft.com/en-us/library/zsyt68f1.aspx>`__ have a non-trivial lifecycle that can make developing and debugging difficult. Web Server controls allow you to add functionality to the client Document Object Model (DOM) elements by using a client control. Tag Helpers have no DOM. Web Server controls include automatic browser detection and some have the ability to pass events from a nested control (such as a button in a table) to the container control.
+
+- Multiple Tag Helpers can act on the same element (see `Avoiding Tag Helper conflicts <http://mvc.readthedocs.org/en/latest/views/tag-helpers/authoring.html#avoiding-tag-helper-conflicts>`__ ) while you typically can't compose Web Server controls.
+
+- Tag Helpers can modify the tag and content of HTML elements, but they don't directly modify anything else on a page. Web Server controls can include CSS stylesheets and javascript libraries, and they can bring in other assets that might have side effects on your page.
+
+- Web Server controls use type converters to convert strings into objects. With Tag Helpers, you work natively in C#, so you don't need to do type conversion. 
+
+- Web Server controls use `System.ComponentModel <https://msdn.microsoft.com/en-us/library/system.componentmodel%28v=vs.110%29.aspx>`__ to implement the run-time and design-time behavior of components and controls. ``System.ComponentModel`` includes the base classes and interfaces for implementing attributes and type converters, binding to data sources, and licensing components. Contrast that to Tag Helpers, which typically derive from ``TagHelper``, and the ``TagHelper`` base class exposes only two methods, ``Process`` and ``ProcessAsync``.
+
+Customizing the Tag Helper element font
+---------------------------------------------
+
+You can customize the font and colorization from **Tools** > **Options** > **Environment** > **Fonts and Colors**:
+
+.. image:: intro/_static/fontoptions.png
+
+.. TODO Move "Tag Helper naming conventions" to the authoring TH doc.
```

Comment:
If you don't want to move it as of yet we should just remove it temporarily. It doesn't fit in this article at all.